js/index.js

```diff
@@ -28,6 +30,26 @@ function bindToForm(form) {
 	form.addEventListener('keyup', handleFormChanges)
 }
 
+const openFiscaRequestBuilder = (form) => () => {
```

Comment:
I like this style of declaration for functions returning functions. 
I've removed the block in the inner function though, it's now inline.